cv_next/hooks/supabaseHooks.ts

```diff
@@ -0,0 +1,28 @@
+import { useContext } from "react"
```

Comment:
if it's under `/hooks` folder, I don't think we need to name the file `Hooks` :)